webapp-django/crashstats/base/jinja2/400.html

```diff
@@ -0,0 +1,43 @@
+{% extends "crashstats_base.html" %}
+
+{% block page_title %}Bad Request{% endblock %}
+
+{% block product_nav_filter %}&nbsp;{% endblock %}
+
+{% block site_css %}
+{{ super() }}
+<style type="text/css">
+ul.errorlist li {
+    list-style: unset;
+    margin-left: 30px;
+}
+.body p {
+    color: red;
+}
+</style>
+{% endblock %}
+
+{% block content %}
+<div id="mainbody">
+    <div class="page-heading">
+        <h2>Bad Request</h2>
+    </div>
+    <div class="panel">
+        <div class="title">
+            <h2>The request's parameters could not be understood.</h2>
+        </div>
+        <div class="body">
+            {% if 'ul class="errorlist"' in error %}
+                {{ error | safe }}
```

Comment:
Are you sure this is really safe? Could you add tests that verify it can't lead to an XSS attack for example?